packages/page-bounties/src/hooks/useBounties.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017-2021 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { SubmittableExtrinsic } from '@polkadot/api/types';
+
+import BN from 'bn.js';
+
+import { DeriveBounties } from '@polkadot/api-derive/types';
+import { useApi, useCall } from '@polkadot/react-hooks';
+import { BalanceOf, BlockNumber } from '@polkadot/types/interfaces';
+
+export type BountyApi = {
+  bestNumber?: BlockNumber,
+  bounties?: DeriveBounties,
+  bountyDepositBase: BN,
+  bountyValueMinimum: BN,
+  dataDepositPerByte: BN,
+  maximumReasonLength: number,
+  proposeBounty: ((...args: any[]) => SubmittableExtrinsic<'promise'>);
+};
+
+export function useBounties (): BountyApi {
+  const { api } = useApi();
+  const bounties = useCall<DeriveBounties>((api.derive.bounties || api.derive.treasury).bounties);
```

Comment:
```suggestion
  const bounties = useCall<DeriveBounties>(api.derive.bounties.bounties);
```

Since we known the API we are working with here.